sphinxcontrib/confluencebuilder/config/defaults.py

```diff
@@ -83,6 +86,7 @@ def apply_defaults(conf):
         'confluence_root_homepage',
         'confluence_watch',
         'singleconfluence_toctree',
+        'confluence_disable_appending_rest_api_to_url'
```

Comment:
(optional) Configuration should be lexicographically sorted against against all configuration entries here.

(note) Any entry here should always have a comma at the end of its entry (even if its at the end).